blog/20231125-simplex-chat-v5-4-link-mobile-desktop-quantum-resistant-better-groups.md

```diff
@@ -9,41 +9,97 @@ imageWide: true
+
+The downside of this approach is that mobile device has to be connected to the same local network. But the upside is that the connection is secure, and you do not need to have a copy of all your data on desktop, which usually has lower security than mobile.
+
+Please note, that the files you send, save or play from desktop app, and also images you view are automatically saved on your desktop device - with the exception of videos, they are locally encrypted in case mobile app has local encryption enabled (which is the default). To remove all these files you can unlink the paired mobile device from the desktop app settings – there will be an option soon allowing to remove the files without unlinking the mobile.
+
+## Better groups
+
+### Faster to join, with more reliable message delivery
+
+We improved the protocols for groups, by making joining groups much faster, and also by adding message forwarding. Previously, the problem was that until a new member connects directly with each existing group member, they did not see each other messages in the group. The problem is explained in detail in [this video](https://www.youtube.com/watch?v=7yjQFmhAftE&t=1104s) at 18:23.
+
+With v5.4, the admin who added members to the group forwards messages to and from the new members until they connect to the existing members. So you should no longer miss any messages and be surprised with replies to messages you have never seen once you and new group members upgrade.
+
+### Create groups with incognito profile
+
+Previously, you could only create groups with your main profile. You could, as a workaround, add yourself to this group with an incognito profile, make this new member an owner, and then group creator could leave the group. This version allows creating groups with incognito profile directly. You will not be able to add your contacts, they can only join via group link.
+
+### Block group members to reduce noise.
 
-## ⚡️ Quick start: use profiles in SimpleX Chat mobile from desktop app
+You now can block messages from group members that send too many messages, or the messages you don't won't to see. It won't affect other group members, and blocked members won't know that you blocked their messages. When they send messages they will appear in the conversation as one line, showing how many messages were blocked. You can reveal them, or delete all sequential blocked messages at once.
 
-## What's the problem?
+### Prohibit files and media in a group
 
-Currently you cannot use the same SimpleX Chat profile on mobile and desktop devices. Even though you can use small groups instead of direct conversations as a workaround, it is quite inconvenient – read status and delivery receipts become much less useful.
+Group owners now have an option to prohibiting sending files and media. This can be useful if you don't won't any images shared, and only want to allow text messages.
```

Comment:
```suggestion
Group owners now have an option to prohibit sending files and media. This can be useful if you don't won't any images shared, and only want to allow text messages.
```